File: src/main/java/org/noxet/noxetserver/messaging/WarningMessage.java
package org.noxet.noxetserver.messaging;

import net.md_5.bungee.api.ChatColor;

public class WarningMessage extends Message {
    public WarningMessage(String text) {
        super(text);
    }

    @Override
    public ChatColor getDefaultColor() {
        return ChatColor.YELLOW;
    }

    @Override
    public String getPrefix() {
        return super.getPrefix() + "§c§lWARNING: ";
    }
}

File: src/main/java/org/noxet/noxetserver/messaging/WarningMessage.scala
package org.noxet.noxetserver.messaging;

import net.md_5.bungee.api.ChatColor;

class WarningMessage(text: String) extends Message(text):
    override def getDefaultColor: ChatColor = ChatColor.YELLOW
    override def getPrefix: String = super.getPrefix + "§c§lWARNING: "